src/Sound.js

```diff
@@ -112,24 +112,28 @@ export default function Sound(props) {
         </button>
       )}
       {sound.previews && (
-        <Waveform
-          waveColor="#FBDC57"
-          backgroundColor="black" barWidth={1}
-          cursorColor="white"  cursorWidth={1}
-          onFinish={() => setIsPlaying(false)}
-          playing={isPlaying}
-          src={sound.previews["preview-lq-mp3"]}
-        />
+        <React.Fragment>
```

Comment:
nit: How about `<>...</>` shorthand?

Reply:
Hmm... Never knew about this! I'll try it out.